source/indexes.txt

```diff
@@ -22,10 +22,10 @@ Optimize Queries by Using Indexes
    :titlesonly:
    :maxdepth: 1
    
-   /indexes/single-field-index
-   /indexes/compound-index
-   /indexes/multikey-index
-   /indexes/atlas-search-index
+   Single Field Indexes </indexes/single-field-index>
+   Compound Indexes </indexes/compound-index>
+   Multikey Indexes </indexes/multikey-index>
+   Atlas Search Indexes </indexes/atlas-search-index>
```

Comment:
```suggestion
   Single Field </indexes/single-field-index>
   Compound </indexes/compound-index>
   Multikey </indexes/multikey-index>
   Atlas Search </indexes/atlas-search-index>
```
S: New discussion with Sara; we can remove indexes if we think it makes sense. Since there are solely index type pages in this section, think it makes sense.